magento/magento2
# Backend Module Manager disable icon fix.

### Description (*)

Module Manager Disable Module icon doesn't display proper.
Magento 2.3 Develop

### Manual testing scenarios (*)

1. Loign to Backend -> system -> Web Setup Wizard -> Module Manager
2. Disable any module and check disable module icon 

### Actual Result (*)
![actualscreenshot](https://user-images.githubusercontent.com/33230237/52856005-2c78d200-3149-11e9-8105-45ebb1d76c00.png)


### Expactced Result (*)
![expectedscreenshot](https://user-images.githubusercontent.com/33230237/52855996-2682f100-3149-11e9-8ff6-5e3a2fe5e256.png)

### Contribution checklist (*)
